.travis/test_cases/test_cache_variables/Design.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Would be nice if these test Design files had some comments added describing _what_ they prove. I mean, if this test passes (pass == build completes successfully?) then how much does that tell us about cachevariables?

Reply:
Yes, I added documentation to a file called 'description.txt' which lives just along the respective Design.